modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/pending_commerce_orders/new_view.jsp

```diff
@@ -9,6 +9,8 @@
 
 <%
 CommerceOrder commerceOrder = commerceOrderContentDisplayContext.getCommerceOrder();
+
+boolean hasPermission = commerceOrderContentDisplayContext.hasModelPermission(commerceOrder, ActionKeys.UPDATE);
```

Comment:
this shoul go below where you use it the first time

Reply:
I repeated a pattern that was already used in `general.jsp` for `commerce-order-web` and that was previously validated by Brian.